app/views/internal/configs/show.html.erb

```diff
@@ -1,238 +1,285 @@
-                           class: "form-control",
-                           value: SiteConfig.periodic_email_digest_min,
-                           placeholder: 2 %>
-        <div class="alert alert-info">Determines the mininum for the periodic email digest</div>
+      <div class="card mt-3">
+        <div class="card-header">Tags</div>
+        <div class="card-body">
+          <div class="form-group">
+            <%= f.label :suggested_tags %>
+            <%= f.text_field :suggested_tags,
+                             class: "form-control",
+                             value: SiteConfig.suggested_tags.join(","),
+                             placeholder: "List of valid tags: comma separated, letters only e.g. beginners,javascript,ruby,swift,kotlin" %>
+            <div class="alert alert-info">Determines which tags are suggested to new users during onboarding (comma
+              separated, letters only)</div>
+          </div>
+        </div>
       </div>
-    </div>
-  </div>
 
-  <div class="card mt-3">
-    <div class="card-header">Tags</div>
-    <div class="card-body">
-      <div class="form-group">
-        <%= f.label :suggested_tags %>
-        <%= f.text_field :suggested_tags,
-                         class: "form-control",
-                         value: SiteConfig.suggested_tags.join(","),
-                         placeholder: "List of valid tags: comma separated, letters only e.g. beginners,javascript,ruby,swift,kotlin" %>
-        <div class="alert alert-info">Determines which tags are suggested to new users during onboarding (comma separated, letters only)</div>
+      <div class="form-group mt-3">
+        <%= f.submit "Update Site Configuration", class: "btn btn-primary" %>
       </div>
+      <% end %>
     </div>
   </div>
+</div>
 
-  <div class="form-group mt-3">
-    <%= f.submit "Update Site Configuration", class: "btn btn-primary" %>
+<div class="row my-3" id="siteConfig">
+  <div class="card w-100">
+    <div class="card-header" id="appConfigHeader">
+      <h2 class="d-inline">Environment variables / AppConfig (readonly)</h2>
+      <button class="btn btn-secondary float-right" type="button" data-toggle="collapse"
+        data-target="#appConfigBodyContainer" aria-expanded="true" aria-controls="appConfigBodyContainer">
+        Toggle
+      </button>
+    </div>
+    <div id="appConfigBodyContainer" class="collapse show hide" aria-labelledby="appConfigHeader">
+      <table class="table">
+        <thead>
+          <tr>
+            <th>Variable</th>
+            <th>Value</th>
+          </tr>
+        </thead>
+        <tbody>
+          <% ENVied.config.variables.each do |v| %>
+          <% if v.group.in?(ENVied.required_groups) %>
+          <tr>
+            <td><%= v.name %></td>
+            <td><%= ApplicationConfig[v.name] %></td>
+          </tr>
+          <% end %>
+          <% end %>
+        </tbody>
+      </table>
+    </div>
   </div>
-<% end %>
+</div>
+<//div>
```

Comment:
Think this should be `</div>` Although it renders fine anyway 🙃

Reply:
Checking again this whole `div` seems to have snuck in, probably some autocomplete mess up.

BTW, I won't get offended if you fix typos etc. on "my" branches yourself, especially if it means we can get things merged.